shell/src/main/java/alluxio/cli/fs/command/SetTtlCommand.java

```diff
@@ -37,8 +37,13 @@
 
   private static final String TTL_ACTION = "action";
 
-  private static final Option TTL_ACTION_OPTION = Option.builder(TTL_ACTION).required(false)
-      .numberOfArgs(1).desc("Action to take after Ttl expiry").build();
+  private static final Option TTL_ACTION_OPTION =
+      Option.builder()
+          .longOpt(TTL_ACTION)
+          .required(false)
+          .numberOfArgs(1)
+          .desc("Action to take after Ttl expiry. Delete (default) or free the target")
```

Comment:
while we're fixing this, Ttl -> TTL here and in getDescription

Reply:
Fixed